Documentation/contributing/testing/ci.rst

```diff
@@ -194,29 +194,27 @@ illustrating which subset of tests the job runs.
 +----------------------------------------------------------------------------------------------------------------+-------------------+--------------------+
 | Jenkins Job                                                                                                    | Trigger Phrases   | Required To Merge? |
 +================================================================================================================+===================+====================+
-| `K8s-1.20-kernel-4.9 <https://jenkins.cilium.io/job/Cilium-PR-K8s-newest-kernel-4.9/>`_                        | test-me-please,   | Yes                |
+| `Cilium-PR-K8s-1.19-kernel-4.9 <https://jenkins.cilium.io/job/Cilium-PR-K8s-1.19-kernel-4.9/>`_                | test-me-please,   | Yes                |
```

Comment:
This doesn't look right as well, `test-me-please` runs k8s 1.20 + kernel 4.9, k8s 1.19 + kernel 4.19 and k8s 1.13 + kernel net-next